cleanlab/count.py

```diff
@@ -56,23 +56,12 @@
 def num_label_issues(
     labels,
     pred_probs,
+    *,
     confident_joint=None,
+    estimation_method="off_diagonal",
 ) -> int:
-    """Estimates the number of label issues in the `labels` of a dataset.
-
-    This method is **more accurate** than ``sum(find_label_issues())`` because its computed using only
-    the trace of the confident joint, ignoring all off-diagonals (which are used by `find_label_issues` and are harder to
-    estimate). Here, we sum over only diagonal elements in the joint (which have more data
-    are more constrained, and therefore easier to compute).
-
-    TL;DR: use this method to get the most accurate estimate of number of label issues when you
-    don't need the indices of the label issues.
-
-    You can use this method to label issues by using `num_label_issues` as the cutoff threshold
```

Comment:
@ulya-tkch You could move some of these descriptions under the docstring describing the argument ``'off_diagonal_recalibrated'``

Reply:
I omitted because was unsure if we should call it the "most accurate estimate" after the benchmarking. Will move back in under `off_diagonal_recalibrated`

Reply:
ya let's not call it most accurate, just say it is another way to estimate that adjusts for under-estimation bias in the other method which limits itself to only count the confident subset of examples